src/index.ts

```diff
@@ -177,14 +195,21 @@ export async function install(
 
   const depObject = {};
   const importMap = {};
+  const styles = [];
   const skipFailures = !isExplicit;
   for (const dep of arrayOfDeps) {
     try {
       const depName = getWebDependencyName(dep);
-      const depLoc = resolveWebDependency(dep, isExplicit, isOptimized);
-      depObject[depName] = depLoc;
-      importMap[depName] = `./${depName}.js`;
-      detectionResults.push([dep, true]);
+      const { depLoc, styleLoc } = resolveWebDependency(dep, isExplicit, isOptimized);
+      if (depLoc) {
+        depObject[depName] = depLoc;
+        importMap[depName] = `./${depName}.js`;
+        detectionResults.push([dep, true]);
+      }
+      if (isExplicit && styleLoc && fs.existsSync(styleLoc)) {
```

Comment:
- We currently do the isExplicit checks inside of `resolveWebDependency`, lets move this check into that function as well
- I don't believe you need `fs.existsSync(styleLoc)` here, since it's presence was checked in resolveWebDependency via the original call to `fs.stat()`